File: PHP/Tarjeta.php
<?php
class Tarjeta extends Payment{
    public $number;
    public $cvv;
    public $date;

    function __construct($id, $number, $cvv, $date) {
        parent::__construct($id);
        $this->number = $number;
        $this->cvv = $cvv;
        $this->date = $date;
    }
}

File: Python/tarjeta.py
from payment import Payment
class Tarjeta(Payment):
    number  = str
    cvv     = str
    date    = str

    def __init__(self, id, number, cvv, date):
        super().__init__(id)
        self.number = number
        self.cvv = cvv
        self.date = date